src/components/10-atoms/toggle-switch/index.scss

```diff
@@ -60,4 +60,15 @@ $toggle-switch-circle-size: $toggle-switch-height - 2 * $toggle-switch-spacing;
       transition: $toggle-switch-transition;
     }
   }
+
+  &__error-message {
+    display: none;
+    
+    &-active {
+      display: block;
+      @include text(t4);
+      color: $color-shy-tomato;
+      margin: 8px 0px 0px;
```

Comment:
If you have the value `0`, it does not matter which unit.

0 potatoes are the same as 0 hamburgers. You you can leave out the unit.

Also, the 3rd parameter looks useless, or is there a reason for that?

Just write: `margin: 8px 0`

Clean and pure.

Reply:
We don't need the extra margin below the text, thats why it is `margin: 8px 0 0`.